### PR TITLE
node-e2e testing: fix alias for stack protector kernel config.

### PR DESCRIPTION
**Pull request by jiahuif**
**What type of PR is this?**
/kind bug
**What this PR does / why we need it**:

The current kernel config validation of `STACKPROTECTOR` has multiple issues:
- There is a YAML syntax problem
 ```yaml
    - CONFIG_CC_STACKPROTECTOR_REGULAR
      CONFIG_CC_STACKPROTECTOR_ALL
```
will be concat'ed to `- CONFIG_CC_STACKPROTECTOR_REGULAR CONFIG_CC_STACKPROTECTOR_ALL`

- There is no need for `CONFIG` prefix, as the validator will add it anyway.

https://github.com/kubernetes/system-validators/blob/master/validators/kernel_validator.go#L141-L143

- `CONFIG_CC_STACKPROTECTOR_STRONG` and its two variants were removed in kernel 4.18, `CONFIG_STACKPROTECTOR_STRONG` [was added to replace it](https://github.com/torvalds/linux/commit/050e9baa9dc9fbd9ce2b27f0056990fc9e0a08a0).

**Special notes for your reviewer**:
This fix should be merged before node e2e tests runs on any kernel >= 4.18 on GKE.
**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

File: test/e2e_node/system/specs/gke.yaml [hunk 1]
```diff
@@ -167,10 +167,11 @@ kernelSpec:
     description: 'Enabled the SECCOMP application API.'
   - name: SECURITY_APPARMOR
     description: 'Enable for AppArmor support.'
-  - name: CC_STACKPROTECTOR_STRONG
+  - name: CC_STACKPROTECTOR_STRONG # until 4.17
```
Comment (mattjmcnaughton) [inline test/e2e_node/system/specs/gke.yaml]: Would you mind making it clear in your comments that the version numbers here refer to linux kernel versions?

**Review by mattjmcnaughton — COMMENTED** [review]
/ok-to-test

This diff looks good to me... but I must admit, fairly far outside my comfort zone. I'll let someone else sign off :)

File: test/e2e_node/system/specs/gke.yaml (hunk 1 above)
Comment (jiahuif) [reply test/e2e_node/system/specs/gke.yaml]: That is a good idea. Thanks!

**Comment by jiahuif** [comment]
For reference
```go
// KernelConfig defines one kernel configuration item.
type KernelConfig struct {
	// Name is the general name of the kernel configuration. It is used to
	// match kernel configuration.
	Name string `json:"name,omitempty"`
	// TODO(yguo0905): Support the "or" operation, which will be the same
	// as the "aliases".
	//
	// Aliases are aliases of the kernel configuration. Some configuration
	// has different names in different kernel version. Names of different
	// versions will be treated as aliases.
	Aliases []string `json:"aliases,omitempty"`
	// Description is the description of the kernel configuration, for example:
	//  * What is it used for?
	//  * Why is it needed?
	//  * Who needs it?
	Description string `json:"description,omitempty"`
}
```

**Pull request merged**